src/privacy/Privacy.js

```diff
@@ -1,7 +1,8 @@
 import {
   getCookie as getCookiePrivate,
-  getCookieBoolean as getCookieBooleanPrivate,
```

Comment:
I moved these over to the library and checked.. yeah, the getCookieBoolean has the type check (as well as the value check), so I think we should use.